tokio-postgres/src/client.rs

```diff
@@ -480,8 +490,13 @@ impl Client {
                     return;
                 }
 
+                let query = if let Some(sp) = self.savepoint {
+                    format!("ROLLBACK TO {}", sp.name)
+                } else {
+                    "ROLLBACK".to_string()
+                };
```

Comment:
nit: this'd be cleaner as a match IMO

Reply:
I matched style with the other instances (e.g https://github.com/sfackler/rust-postgres/blob/64caf4c947cfde6c292f42406c046dce79f771bf/tokio-postgres/src/transaction.rs#L42-L46 and https://github.com/sfackler/rust-postgres/blob/64caf4c947cfde6c292f42406c046dce79f771bf/tokio-postgres/src/transaction.rs#L83-L87). Would you like to switch all of those or just this one?

Reply:
Oh huh, let's keep it as-is then.